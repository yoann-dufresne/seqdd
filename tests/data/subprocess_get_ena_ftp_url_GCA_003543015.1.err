  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100  3586    0  3586    0     0  36907      0 --:--:-- --:--:-- --:--:-- 36969
