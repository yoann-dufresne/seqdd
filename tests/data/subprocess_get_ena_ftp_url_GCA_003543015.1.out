<?xml version="1.0" encoding="UTF-8"?>
<ASSEMBLY_SET>
<ASSEMBLY accession="GCA_003543015.1" alias="ASM354301v1" center_name="University of Queensland">
  <IDENTIFIERS>
    <PRIMARY_ID>GCA_003543015.1</PRIMARY_ID>
    <SUBMITTER_ID namespace="University of Queensland">ASM354301v1</SUBMITTER_ID>
  </IDENTIFIERS>
  <TITLE>ASM354301v1 assembly for Micromonosporaceae bacterium</TITLE>
  <NAME>ASM354301v1</NAME>
  <ASSEMBLY_LEVEL>scaffold</ASSEMBLY_LEVEL>
  <GENOME_REPRESENTATION>full</GENOME_REPRESENTATION>
  <TAXON>
    <TAXON_ID>1873464</TAXON_ID>
    <SCIENTIFIC_NAME>Micromonosporaceae bacterium</SCIENTIFIC_NAME>
  </TAXON>
  <SAMPLE_REF>
    <IDENTIFIERS>
      <PRIMARY_ID>SAMN08019851</PRIMARY_ID>
    </IDENTIFIERS>
  </SAMPLE_REF>
  <STUDY_REF>
    <IDENTIFIERS>
      <PRIMARY_ID>PRJNA417962</PRIMARY_ID>
    </IDENTIFIERS>
  </STUDY_REF>
  <WGS_SET>
    <PREFIX>DPJM</PREFIX>
    <VERSION>01</VERSION>
  </WGS_SET>
  <ASSEMBLY_LINKS>
    <ASSEMBLY_LINK>
      <URL_LINK>
        <LABEL>WGS_SET_FLATFILE</LABEL>
        <URL>ftp://ftp.ebi.ac.uk/pub/databases/ena/wgs/public/dpj/DPJM01.dat.gz</URL>
      </URL_LINK>
    </ASSEMBLY_LINK>
    <ASSEMBLY_LINK>
      <URL_LINK>
        <LABEL>WGS_SET_FASTA</LABEL>
        <URL>ftp://ftp.ebi.ac.uk/pub/databases/ena/wgs/public/dpj/DPJM01.fasta.gz</URL>
      </URL_LINK>
    </ASSEMBLY_LINK>
  </ASSEMBLY_LINKS>
  <ASSEMBLY_ATTRIBUTES>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>total-length</TAG>
      <VALUE>4351932</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>ungapped-length</TAG>
      <VALUE>4320771</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>n50</TAG>
      <VALUE>5359</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>spanned-gaps</TAG>
      <VALUE>99</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>unspanned-gaps</TAG>
      <VALUE>0</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>scaffold-count</TAG>
      <VALUE>861</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>count-contig</TAG>
      <VALUE>960</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>contig-n50</TAG>
      <VALUE>5170</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>contig-L50</TAG>
      <VALUE>275</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>contig-n75</TAG>
      <VALUE>3557</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>contig-n90</TAG>
      <VALUE>2829</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>scaf-L50</TAG>
      <VALUE>266</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>scaf-n75</TAG>
      <VALUE>3723</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>scaf-n90</TAG>
      <VALUE>2971</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>replicon-count</TAG>
      <VALUE>0</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>count-non-chromosome-replicon</TAG>
      <VALUE>0</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>count-alt-loci-units</TAG>
      <VALUE>0</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>count-regions</TAG>
      <VALUE>0</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>count-patches</TAG>
      <VALUE>0</VALUE>
    </ASSEMBLY_ATTRIBUTE>
    <ASSEMBLY_ATTRIBUTE>
      <TAG>ENA-LAST-UPDATED</TAG>
      <VALUE>2018-09-10</VALUE>
    </ASSEMBLY_ATTRIBUTE>
  </ASSEMBLY_ATTRIBUTES>
</ASSEMBLY>
</ASSEMBLY_SET>
